mne/utils/dataframe.py

```diff
@@ -62,17 +63,21 @@ def _build_data_frame(inst, data, picks, long_format, mindex, index,
     for i, (k, v) in enumerate(mindex):
         df.insert(i, k, v)
     # build Index
+    kwargs = dict()
+    key = 'copy' if 'copy' in _get_args(df.set_index) else 'inplace'
+    val = False if key == 'copy' else True
+    kwargs[key] = val
     if long_format:
-        df.set_index(default_index, inplace=True)
+        df = df.set_index(default_index, **kwargs)
```

Comment:
this won't work because `df.set_index` will return `None` when `inplace=True`.  I'll fix it.

Reply:
Yeah, this is true on old Pandas at lea,st. When `copy=False` it actually returns an instance now, and you *must* use it (the original is not in the correct state anymore). I pushed a commit to fix it already